### DISCUSSION TITLE
Enterprise Open Source A Practical Introduction

### DISCUSSION THREAD

**Discussion by wangleo61** [kaiyuanshe/ONES#42] [案例分享 & 最佳实践]
企业界的开源实践一起来了解一下哈，有兴趣的一起么
[Uploading Enterprise Open Source A Practical Introduction.pdf…]()

**Comment by will-ww** [comment]
太有了，我看到不少材料中的内容，都是来自这篇文章中的内容改写的，可以详细讨论起来，例如下次的 ONES Group 双周会上一起分享下~

这个链接好像有问题，我又上传了一份：[下载](https://www.linuxfoundation.org/wp-content/uploads/lfpub_enterprise_os_practical_introduction_080818.pdf#page=16)

[lfpub_enterprise_os_practical_introduction_080818.pdf](https://github.com/kaiyuanshe/ONES/files/8259441/lfpub_enterprise_os_practical_introduction_080818.pdf)

**Reply by will-ww** [reply]
已添加到双周讨论会提议中：https://github.com/kaiyuanshe/ONES/issues/44
